composer/soso-richtext.html

```diff
@@ -385,6 +386,32 @@
           let n = nodes[i];
           n.removeAttribute("style");
         }
+        nodes = div.querySelectorAll('[class]');
+        for (var i = 0; i < nodes.length; i++) {
+          let n = nodes[i];
+          n.removeAttribute("class");
+        }
+      }
+
+      _cleanCusomElements(div) {
+        let nl = div.childNodes;
+        if (nl && nl.length) {
+          for (let i = 0; i < nl.length; i++) {
+            let n = nl[i];
+            if (n.nodeType === Node.ELEMENT_NODE) {
+              let tagName = n.tagName;
+              if (tagName) {
+                if (tagName.indexOf('-') >= 0) {
+                  let p = document.createElement("p");
+                  p.textContent = n.textContent || "";
```

Comment:
Isn't there going to be a problem with things like <BR> and inner <span> and <div> where the space between words gets lost?

Reply:
Yes there may be cases but this is strictly for custom elements where it is hard to predict what might be inside them and how it's rendered. There are cases like, in firefox even normal paste messes up html layout. At least all text is preserved and user may reformat it. 

This is mainly for copying from own articles.